app/assets/src/components/ui/controls/tooltips/BetaLabel.jsx

```diff
@@ -0,0 +1,12 @@
+import { Label as BaseLabel } from "semantic-ui-react";
```

Comment:
Can you move this one level down, i.e. `ui/tooltips`? 
Maybe just call it labels, since this is not necessarily something that appears on interaction/hover.

Reply:
ok